src/index.js

```diff
@@ -10,19 +10,23 @@ require('brace/mode/storygen');
 var math = require('mathjs');
 var Clipboard = require('clipboard');
 
+// Parses dictionary entry from line
```

Comment:
Please try to avoid tautological comments. This one adds nothing to function name.